### ISSUE TITLE
Option to include shortcuts in list of frequent apps

### ISSUE THREAD

**Issue by DBecker7** [Ulauncher/Ulauncher#678]
### Describe the solution you'd like

When I hit Ctrl+Space, I would like to see my most frequent shortcuts along with most frequent apps. The things I see when I hit Ctrl+Space should reflect the way I use ULauncher.

I think this should be an option rather than the default since other people have different workflows.

### Additional context

Ulauncher's killer feature (for me) is the shortcuts. My Atom/VSCodium/RStudio projects are assigned to folders, and adding shortcuts like "atom ~/myproj" is a wonderful way to switch right into a project.

**Comment by gornostal** [comment]
Yep, it would be a nice feature to have.
If anyone is interested in implementing it, post a comment here.

**Comment by nishp77** [comment]
@DBecker7 want to ask you that currently we can only see frequently used apps but it doesn't show the shortcuts, and that's what you want? Yes ?

**Comment by DBecker7** [comment]
That's correct! It would be nice for the shortcuts to show up as suggestions before I've even typed anything.
